### PR TITLE
Code field for BRK2 KGE and KCE.

### PR DESCRIPTION
**Pull request by peteradrichem**

### CODE REVIEW

File: src/data/sql/brk2/select.kadastrale_gemeente.sql [hunk 1]
```diff
@@ -4,23 +4,29 @@
...
+                         aangeduid_door_brk_kadastralegemeente_code         AS code
                   FROM brk2_prep.kadastraal_object
                   WHERE indexletter = 'G'
                     AND ST_IsValid(geometrie)
                     AND datum_actueel_tot IS NULL) gc1
             WHERE gc1.nrings[1] = 0
             GROUP BY gc1.identificatie,
-                     gc1.ligt_in_gemeente) gc2) gc3
+                     gc1.ligt_in_gemeente,
+                     gc1.code) gc2) gc3
```
Comment (JKoetsier) [inline src/data/sql/brk2/select.kadastrale_gemeente.sql:29]: Have you checked the result of this query with the result of the original query? Are we getting the same number of rows back?

File: src/data/sql/brk2/select.kadastrale_gemeentecode.sql [hunk 2]
```diff
@@ -4,23 +4,29 @@
...
+                         aangeduid_door_brk_kadastralegemeentecode_code         AS code
                   FROM brk2_prep.kadastraal_object
                   WHERE indexletter = 'G'
                     AND ST_IsValid(geometrie)
                     AND datum_actueel_tot IS NULL) gc1
             WHERE gc1.nrings[1] = 0
             GROUP BY gc1.identificatie,
-                     gc1.is_onderdeel_van_kadastralegemeente) gc2) gc3
+                     gc1.is_onderdeel_van_kadastralegemeente,
+                     gc1.code) gc2) gc3
```
Comment (JKoetsier) [inline src/data/sql/brk2/select.kadastrale_gemeentecode.sql:29]: Same here. Are we getting the same number of rows back?

Comment (peteradrichem) [reply src/data/sql/brk2/select.kadastrale_gemeentecode.sql:29]: The result is 86 rows for both queries.

**Review by jeroenriemens — APPROVED** [review]
Mooi

**Review by JKoetsier — DISMISSED** [review]

**Pull request merged**